HM_05.1/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HM_05._1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] array = new int[3];
+            for (int i = 0; i < array.Length - 1; i++)
+            {
+                Console.WriteLine("Введите элементы массива");
+                array[i] = int.Parse(Console.ReadLine());   
+            }
+            for (int i2 = 0; i2 < array.Length - 1; i2++)
```

Comment:
you **do not need** to use different name of the **for** iterator if you have some **for** operators in your code.

for each **for** operator you can just use **i** instead of **i, i2, i3**